pkgs/tools/system/swaysettings/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  meson,
```

Comment:
Commas come before the arguments, nor after them.

```suggestion
  , meson
```

Reply:
Ok corrected